pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -675,37 +674,6 @@ func getCachedRepoInfo(upgrade *harvesterv1.Upgrade) (*repoinfo.RepoInfo, error)
 	return repoInfo, nil
 }
 
-func isVersionUpgradable(currentVersion, minUpgradableVersion string) error {
```

Comment:
This is unrelated to the issue, but I came across some unused code while tracing the code.